dockernet/scripts/community-pool-staking/create_pool.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -eu 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/../../config.sh
+
+LIQUIDITY=1000000000000
+GAS="--gas-prices 0.1uosmo --gas auto --gas-adjustment 1.3"
+
+echo "Determining relevant channels..."
+dydx_to_osmo_client=$(GET_CLIENT_ID_FROM_CHAIN_ID DYDX OSMO)
+dydx_to_osmo_connection=$(GET_CONNECTION_ID_FROM_CLIENT_ID DYDX $dydx_to_osmo_client)
+dydx_to_osmo_channel=$(GET_TRANSFER_CHANNEL_ID_FROM_CONNECTION_ID DYDX $dydx_to_osmo_connection)
+osmo_to_dydx_channel=$(GET_COUNTERPARTY_TRANSFER_CHANNEL_ID DYDX $dydx_to_osmo_channel)
+dydx_denom_on_osmo=$(GET_IBC_DENOM OSMO $osmo_to_dydx_channel $DYDX_DENOM)
+
+echo -e "\nDYDX -> OSMO:"
+echo "  Client: $dydx_to_osmo_client"
+echo "  Connection: $dydx_to_osmo_connection"
+echo "  Transfer Channel: $dydx_to_osmo_channel -> $osmo_to_dydx_channel"
+echo "  IBC Denom: $dydx_denom_on_osmo"
+
+noble_to_osmo_client=$(GET_CLIENT_ID_FROM_CHAIN_ID NOBLE OSMO)
+noble_to_osmo_connection=$(GET_CONNECTION_ID_FROM_CLIENT_ID NOBLE $noble_to_osmo_client)
+noble_to_osmo_channel=$(GET_TRANSFER_CHANNEL_ID_FROM_CONNECTION_ID NOBLE $noble_to_osmo_connection)
+osmo_to_noble_channel=$(GET_COUNTERPARTY_TRANSFER_CHANNEL_ID NOBLE $noble_to_osmo_channel)
+usdc_denom_on_osmo=$(GET_IBC_DENOM OSMO $osmo_to_noble_channel $USDC_DENOM)
+
+echo -e "\nNOBLE -> OSMO:"
+echo "  Client: $noble_to_osmo_client"
+echo "  Connection: $noble_to_osmo_connection"
+echo "  Transfer Channel: $noble_to_osmo_channel -> $osmo_to_noble_channel"
+echo "  IBC Denom: $usdc_denom_on_osmo"
+
+echo -e "\nSending dydx/usdc to osmosis for initial liquidity..."
+
+echo ">>> DYDX to Osmosis:"
+$DYDX_MAIN_CMD tx ibc-transfer transfer transfer $dydx_to_osmo_channel $(OSMO_ADDRESS) ${LIQUIDITY}${DYDX_DENOM} \
+    --from ${DYDX_VAL_PREFIX}1 -y | TRIM_TX
+
+echo ">>> USDC to Osmosis:"
+$NOBLE_MAIN_CMD tx ibc-transfer transfer transfer $noble_to_osmo_channel $(OSMO_ADDRESS) ${LIQUIDITY}${USDC_DENOM} \
+    --from ${NOBLE_VAL_PREFIX}1 -y | TRIM_TX
+sleep 15
+
+echo ">>> Balances:"
+$OSMO_MAIN_CMD q bank balances $(OSMO_ADDRESS)
+
+echo -e "\nCreating dydx/usdc pool on osmosis..."
+pool_file=${STATE}/${OSMO_NODE_PREFIX}1/pool.json
```

Comment:
what is STATE here?

Reply:
it's an env variable that defines the dockernet state directory (`dockernet/state`)